cli/tests/integration_tests.rs

```diff
@@ -2048,6 +2048,29 @@ itest!(deno_lint_glob {
   exit_code: 1,
 });
 
+#[test]
+fn cafile_env_fetch() {
+  use url::Url;
+  let g = util::http_server();
+  let deno_dir = TempDir::new().expect("tempdir fail");
+  let module_url =
+    Url::parse("http://localhost:4545/cli/tests/cafile_url_imports.ts")
+      .unwrap();
```

Comment:
I think this URL should be https://localhost:5545/cli/tests/cafile_info.ts

I see that cafile_fetch below is also using a http URL rather than https. I think that is an error.

What do you think?

Reply:
@ry Nice catch! The weird thing is that test is actually failing if i remove the `DENO_CERT` env (i tested with and without it before submitting the PR) so something weird is happening? 🤷 

I changed to `https://localhost:5545/cli/tests/cafile_info.ts` and it works. Want me to update the `cafile_fetch` test in the same PR?